modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,102 @@
+
+* Labels for `Compute Address` are supported in the GCP beta version. {product-title} does not add labels to the resource.
+
+User-defined labels and {product-title} labels are seen on the following GCP resources:
+
+* Compute Instance
+* Compute Disk
+* Compute Forwarding Rule
+* Compute Image
+* DNS Managed Zone
+* Filestore Instance
+* Storage Bucket
+
+.User-defined tags
+
+User-defined `tags` are attached to resources created by {product-title} core operator `cluster-image-registry-operator` and not on the resources created by any other operators or the kubernetes in-tree components. In {product-title} version {product-version}, resources created by the foloowing operators are not tagged:
+* Installer Operator
+* Machine API Provider GCP operator 
+* GCP Filestore CSI Driver Operator
+* `GCP PD CSI Driver Operator
+
+The following are the requirements for the user-defined tags:
+
+* Tag key and tag value must already exist. {product-title} does not create the key and the value .
+* A tag `parentID` can be either `OrganizationID` or `ProjectID`:
+** An `OrganizationID` must consist of decimal numbers, and cannot have leading zeros.
+** A `ProjectID` must be 6 to 30 characters in length, can only contain lowercase letters, numbers, and hyphens, and must start with a letter, and cannot end with a hyphen.
+* A tag key and value must have a minimum of 1 character and can have a maximum of 63 characters.
+* A tag key must contain only uppercase and lowercase alphanumeric characters, hyphen (`-`), underscore (`_`), and period (`.`).
+* A tag value must contain only uppercase and lowercase alphanumeric characters, hyphen (`-`), underscore (`_`), period (`.`), at sign (`@`), percent sign (`%`), equals sign (`=`), plusse (`+`), colon (`:`), comma (`,`), asterisk (`*`), pound sign (`$`), ampersand (`&`), parentheses (`()`), square braces (`[]`), curly braces (`{}`), and space.
+* A tag key and value must begin and end with an alphanumeric character.
+* Tag value must be one of the pre-defined values for the key.
+* You can configure a maximum of 50 tags.
+* There should be no tag key defined with the same value as any of the existing tag keys that will be inherited from the parent resource.
+
+//Cluster creation phase
+include::installing-gcp-cluster-creation.adoc[leveloffset=+1]
+
+.Limitations for the user-defined tags
+
+* Tags will not be attached to the following items:
+** Control plane instances and storage buckets created by the installation program
+** Compute instances created by the Machine API provider for GCP
+** Filestore instance resources created by the GCP Filestore CSI Driver Operator
+** Compute disk and compute image resources created by the GCP PD CSI Driver Operator
+* Tags are not supported for buckets located in the following regions:
+** `us-east2` 
+** `us-east3`
+* Cluster Image Registry Operator does not throw any error but skips processing tags when the buckets are created in the tags unsupported region.
```

Comment:
I think it's Image Registry Operator. Is there something called Cluster Image Registry Operator? can you confirm?

Reply:
@bharath-b-rh, can you ptal at the newly created modules?

Reply:
@snarayan-redhat "Cluster Image Registry Operator" is official (see [Operator reference](https://docs.openshift.com/container-platform/4.13/operators/operator-reference.html#cluster-image-registry-operator_cluster-operators-ref)), but you are correct that omitting "Cluster" is a very common practice in our docs.

Reply:
@jeana-redhat got that! Thanks for confirming.

Reply:
And feedbaack from bharath: "It's called Image Registry Operator in docs, and cluster-image-registry-operator in deployments."

Reply:
My suggestion would be to keep it Image Registry Operator just for consistency across docs.

Reply:
Ack.